public/app/features/alerting/unified/rule-list/components/ActionsLoader.tsx

```diff
@@ -0,0 +1,5 @@
+import Skeleton from 'react-loading-skeleton';
+
+export function ActionsLoader() {
```

Comment:
nit/naming: The name `actions loader` makes me think it's going to load the actions in for me. Perhaps we could call it `ActionsSkeleton`?

Reply:
Good point. We have a component like `AlertRuleListItemLoader` which also renders Skeleton, but you're right the name is confusing as other `Loaders` actually load data